From 8cdc31cb9c360ba264461aa3a8a365f3c183aed4 Mon Sep 17 00:00:00 2001
From: Josh Wu <josh.wu@atmel.com>
Date: Wed, 19 Feb 2014 17:49:20 +0800
Subject: [PATCH] sama5d3xek: for sama5d35ek board, it doesn't support pda dtb

fix the bug: the system cannot boot when sama5d35ek board is inserted a
PDA module.

Signed-off-by: Josh Wu <josh.wu@atmel.com>
---
 include/configs/sama5d3xek.h | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/include/configs/sama5d3xek.h b/include/configs/sama5d3xek.h
index 4273c6e..16d6752 100644
--- a/include/configs/sama5d3xek.h
+++ b/include/configs/sama5d3xek.h
@@ -199,7 +199,11 @@
 		"if test $dm_type = undefined; then "		\
 			"setenv conf_name $mb_name; "		\
 		"else "						\
-			"setenv conf_name $mb_name'_'$dm_type; "\
+			"if test $ek_name = sama5d35ek; then "	\
+				"setenv conf_name $mb_name; "	\
+			"else "					\
+				"setenv conf_name $mb_name'_'$dm_type; "\
+			"fi; "					\
 		"fi; "						\
 		"setenv fdtfile $conf_name'.dtb'; \0"
 
-- 
1.9.1

